public/base-locales/pl.json

```diff
@@ -85,40 +85,40 @@
   "locale_selection_pt-br": "Português",
   "locale_selection_sv": "Svenska",
   "locale_selection_th": "ไทย",
-  "import": "Importuj",
-  "export": "Eksportuj",
-  "logout": "Wyloguj",
+  "import": "Import ustawień",
+  "export": "Eksport ustawień",
+  "logout": "Wyloguj się",
   "local_storage_cleared": "Lokalna pamięć została wyczyszczona!",
   "iv_or_filter": "IV OR Filtr",
   "polygons": "Obszary",
   "raids": "Raidy",
-  "ex_eligible": "EX Gymy",
+  "ex_eligible": "Gymy EXowe",
   "in_battle": "W trakcie walki",
-  "all_pokestops": "Pokestopy",
-  "all_gyms": "Gymy",
+  "all_pokestops": "Wszystkie Pokestopy",
```

Comment:
![image](https://user-images.githubusercontent.com/10072920/156946515-2f29167e-c861-4c52-998c-154d22127fb9.png)
Wanted to avoid that due to the fact it's taking two lines.

Reply:
It doesn't matter. It's confusing without `wszystkie`